widget/table.go

```diff
@@ -508,7 +508,20 @@ func (r *tableCellsRenderer) MinSize() fyne.Size {
 	} else {
 		fyne.LogError("Missing Length callback required for Table", nil)
 	}
-	return fyne.NewSize(r.cells.cellSize.Width*float32(cols)+float32(cols-1), r.cells.cellSize.Height*float32(rows)+float32(rows-1))
+
+	width := float32(0)
```

Comment:
nit: not sure if this makes a difference to the compiler, but declaring a var seems nicer than casting 0
```suggestion
	var width float32
```

Reply:
If we think there should be a rule for doing it one way or another then probably should also be in the style guide.

Reply:
In the absence of a style guide, reviewers' style is adopted, no?

Reply:
That doesn't really make sense as we can have many reviewers - so there then would be conflicts.
If a PR does not go against our style or standards it should be approved surely?

Reply:
The var declaration would be shorter, and doesn't have a type cast, but whatever